### PR TITLE
Add relatedSource attribute to Animal data model

### PR DESCRIPTION
**Pull request by fdelavega**
#474 #429

### CODE REVIEW

File: specs/AgriFood/Animal/doc/spec.md [hunk 1]
```diff
@@ -29,6 +29,14 @@ The proposed model for the animal entity has the following properties:
...
     -   Allowed values: (dairy cattle, beef cattle, sheep, goat, horse, pig)
     -   Mandatory
+-   relatedSource: ID of the animal in external applications
+    -   Attribute type: List of StructuredValue
+    -   Values:
+        -   application: ID of the particular AgriApp entity
+            -   Type: URI
+        -   applicationEntityId: ID of the animal in the external application
+            -   Type: Text
+    -   Optional
```
Comment (jmcanterafonseca) [inline specs/AgriFood/Animal/doc/spec.md]: I think the common Agrifood data model includes other attributes, doesn't it?

**Comment by jmcanterafonseca** [comment]
@fdelavega can you check the status of this so that we can make progress?

File: specs/AgriFood/Animal/doc/spec.md (hunk 1 above)
Comment (fdelavega) [reply specs/AgriFood/Animal/doc/spec.md]: currently it only includes relatedSource

**Comment by jmcanterafonseca** [comment]
LGTM, landing

**Pull request merged**